storm-client/test/jvm/org/apache/storm/security/auth/AutoLoginModuleTest.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2018 The Apache Software Foundation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.storm.security.auth;
```

Comment:
Since this is really testing `org.apache.storm.security.auth.kerberos.AutoTGTKrb5LoginModule`, can we move the unit test to `storm-client/test/jvm/org/apache/storm/security/auth/kerberos/AutoLoginModuleTest.java`? Thanks